modules/sor/sor-debug.test.ts

```diff
@@ -58,7 +58,7 @@ describe('sor debugging', () => {
             swapType: 'EXACT_IN',
             swapAmount: '0.01',
             useProtocolVersion,
-            poolIds: ['0x2e629cd9061e2b5214b6cfc8d001deb726275eb0'], // gyroECLP
+            poolIds: ['0xCd2Ca3bBe68Fb83E9B1f1217deCF3Fb01dbFb01E'], // gyroECLP
```

Comment:
hey, we store all addresses in lowercase. Could you make it lowercase just to be sure it get's picked up correctly?
```suggestion
            poolIds: ['0xcd2ca3bbe68fb83e9b1f1217decf3fb01dbfb01e'], // gyroECLP
```